File: demo-test/src/test/java/io/fries/demo/test/cucumber/wiremock/MockServers.java
package io.fries.demo.test.cucumber.wiremock;

import java.util.List;

public class MockServers implements MockServer {

    private final List<MockServer> servers;

    public MockServers(final List<MockServer> servers) {
        this.servers = servers;
    }

    public static MockServers none() {
        return new MockServers(List.of());
    }

    @Override
    public void start() {
        servers.forEach(MockServer::start);
    }

    @Override
    public void stop() {
        servers.forEach(MockServer::stop);
    }
}

File: demo-test/src/test/kotlin/io/fries/demo/test/cucumber/wiremock/MockServers.kt
package io.fries.demo.test.cucumber.wiremock

class MockServers(
    private val servers: List<MockServer> = listOf()
) : MockServer {

    override fun start() {
        servers.forEach(MockServer::start)
    }

    override fun stop() {
        servers.forEach(MockServer::stop)
    }
}